### ISSUE TITLE
Jasper says this widget doesn't work in Mx7

### ISSUE THREAD

**Issue by tieniber** [mendix/MicroflowTimer#10]

**Comment by tieniber** [comment]
@simo101 has a version that works in Mx7. Maybe he can submit a PR

**Comment by simo101** [comment]
Here is the version i use that works. I can't remember making any changes. Must be something different?
[MicroflowTimer.mpk.zip](https://github.com/mendix/MicroflowTimer/files/1138344/MicroflowTimer.mpk.zip)

**Comment by JelteMX** [comment]
Yeah some things were changed regarding the `_execMf`

**Comment by JelteMX** [comment]
Hold on, let me try to fix this

**Comment by JelteMX** [comment]
Can you test [this version](https://github.com/mendix/MicroflowTimer/raw/fix_mx_7/test/widgets/MicroflowTimer.mpk) and tell me whether or not the problem is fixed?

**Comment by Mocke44** [comment]
I had the same issue as @JelteMX, when converting from Mx 6.10.3 to Mx 7.5.1.
@simo101 your updated version fixed my issue, thanks.

**Comment by Mocke44** [comment]
I converted a project from 6.10.3 to 7.5.1 and i get the same issue, when checking the browser console i see this : 
mxui.js?636371150751144462:5 Uncaught (in promise) TypeError: Cannot read property 'insertBefore' of null
    at Function.undefined.f.injectUrl (mxui.js?636371150751144462:5)
    at te (mxui.js?636371150751144462:5)
    at c (mxui.js?636371150751144462:5)
    at mxui.js?636371150751144462:5
    at Vt (mxui.js?636371150751144462:5)
    at ft (mxui.js?636371150751144462:5)
    at pt (mxui.js?636371150751144462:5)
    at f (mxui.js?636371150751144462:5)
    at F (mxui.js?636371150751144462:15)
    at mxui.js?636371150751144462:15
undefined.f.injectUrl @ mxui.js?636371150751144462:5
te @ mxui.js?636371150751144462:5
c @ mxui.js?636371150751144462:5
(anonymous) @ mxui.js?636371150751144462:5
Vt @ mxui.js?636371150751144462:5
ft @ mxui.js?636371150751144462:5
pt @ mxui.js?636371150751144462:5
f @ mxui.js?636371150751144462:5
F @ mxui.js?636371150751144462:15
(anonymous) @ mxui.js?636371150751144462:15
B @ mxui.js?636371150751144462:15
startup @ mxui.js?636371150751144462:15
w @ mxui.js?636371150751144462:15
r @ mxui.js?636371150751144462:5
a @ mxui.js?636371150751144462:5

Please advise if there is a fix for this, i have tried both versions of the widget in this thread, neither resolved the issue.

**Comment by JelteMX** [comment]
As there is zero reference to the Microflowtimer widget, I am not entirely sure that your issue is related to this widget. Any other widgets you are using? 

@tieniber have you checked my version already? Need to merge this into the widget

**Comment by Mocke44** [comment]
@JelteMX ,

Sorry for the delayed response. I have done some Investigation and it appears the widget is not the issue. 
I looked into the release notes doc and converting from 6 to 7 and i found the following article :https://docs.mendix.com/releasenotes/desktop-modeler/7.0 . 
Under the heading "Client APIs deprecated since Mendix 5" it mentions that the api "mxui.InsertBefore" has been removed in mx 7, which is the function which throws an error in the browser console.

It seems i must some how update my mxui.js, or is there another fix for this?
